<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QDialog" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/document-export</normaloff>:/document-export</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="settingsBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="formatBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Export items:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="contactsBox">
          <property name="text">
           <string>Contacts</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="messagesBox">
          <property name="text">
           <string>Messages</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="calendarBox">
          <property name="text">
           <string>Calendar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Export directory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="exportLine"/>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="openBrowserButton">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="directoryLabel">
        <property name="text">
         <string>No export directory selected!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="contactBox">
     <property name="title">
      <string>Contact export...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="allContactsButton">
        <property name="text">
         <string>Export all contacts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="messageContactsButton">
        <property name="text">
         <string>Only export contacts with at least one message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="filterButton">
        <property name="text">
         <string>Filter contacts and show only messages from these contacts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="contactsList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="selectAllButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deselectAllButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Deselect All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pluginSettingsBox">
     <property name="title">
      <string>Plugin settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="periodLabel">
          <property name="text">
           <string>Period per file:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="periodBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="orderLabel">
          <property name="text">
           <string>Message order:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="orderBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="graphBox">
        <property name="text">
         <string>Include statistics graph</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="graphFormatLabel">
              <property name="text">
               <string>Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="graphFormatBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="legendBox">
            <property name="text">
             <string>Show legend</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="thumbnailsBox">
        <property name="text">
         <string>Show thumbnail images for contacts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Export</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Export</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>contactsList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectAllButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>deselectAllButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphFormatLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphFormatBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
